### PR TITLE
fix(cvs): fix custom cvs for ntb

### PR DESCRIPTION
**Pull request by petrjasek**
- use `ANPA Category` label to be consistent with superdesk
- set scheme for custom subject so it will show up there in authoring

SDNTB-492

### CODE REVIEW

File: client/services/PlanningStoreService.js [hunk 1]
```diff
@@ -221,3 +220,11 @@ export function PlanningStoreService(
 
     $rootScope.$on('vocabularies:updated', angular.bind(this, this._reloadVocabularies));
 }
+
+function getSubjects(values) {
+    if (values.subject_custom) {
+        return values.subject_custom.map((item) => Object.assign({}, item, {scheme: 'subject_custom'}));
+    }
+
+    return values.subject;
```
Comment (MarkLark86) [inline client/services/PlanningStoreService.js]: Should this be:
`return values.subjectcodes`

File: client/utils/index.js [hunk 2]
```diff
@@ -217,6 +217,7 @@ export const createStore = (params = {}, app = planningApp) => {
     ];
     // return the store
 
+    console.info('state', initialState);
```
Comment (ioanpocol) [inline client/utils/index.js]: Do we need this log?

Comment (petrjasek) [reply client/utils/index.js]: I don't think so :+1:

**Comment by petrjasek** [comment]
seems like there is some issue with genre not being saved atm, but that's also on sdp-master, @MarkLark86 is that known issue?

**Comment by MarkLark86** [comment]
@petrjasek The latest Autosave improvement (save autosave on the server) has been causing multiple issues. We're in the progress of investigating these issues and will be creating tickets shortly

**Pull request merged**